client/src/templates/Challenges/classic/lower-jaw.tsx

```diff
@@ -165,6 +186,18 @@ const LowerJaw = ({
     );
   };
 
+  const challengeOrder: ChallengeWithCompletedNode[] = getChallengesList({
+    userInputSuperBlock: superBlock || '',
+    userInputBlock: block || ''
+  }) as ChallengeWithCompletedNode[];
+  if (challengeOrder[challengeOrder.length - 1].id === lastChallengeId) {
+    endOfProject.current = true;
+  }
+  const showShareButton =
+    challengeIsCompleted &&
+    endOfProject.current &&
+    completedPercent === isBlockCompleted;
```

Comment:
```suggestion
  const showShareButton =
    challengeIsCompleted && completedPercent === isBlockCompleted;
```

Reply:
Hello @Sboonny,

Mainly this share button need to show only at **end of Project** as per requirement 👇. So added the ``endOfProject.current`` logic.

If i remove this logic then if you achieved 100% and if you went to the start or middle of the project then this share button will show. but as per @ahmadabdolsaheb requirement, we need only at the end of the project

I understand if user goes unordered way to complete the challenges then it make sense showing share button at any point of the project, 

Discuss with your team (@ahmadabdolsaheb) about requirement and update the requirement then will remove the logic

<img width="825" alt="image" src="https://user-images.githubusercontent.com/36238382/230704428-7c47a60e-909e-4fbf-aaf1-88f7d4a3b669.png">

Reply:
Updated the spec to read "Add a share button below the progress bar at the project completion".

Reply:
Thanks for your response, will update soon

Reply:
Hey really sorry for not updating, getting some PNPM issues, will try to update in 1 or 2 days. Won't make it much late

Reply:
Take your time, there is no hurry. if you face an issue, tell us in discord or the forum.

Reply:
Hello @Sboonny ,

I updated it code, also some other unit tests are failing to update snapshot, i am updating in local but again those are failing in build, please check on it once